Automattic/wp-calypso
# Add transition to sidebar when selecting a site.

Related to https://github.com/Automattic/dotcom-forge/issues/6765

## Proposed Changes

Add transition when selecting a site, and the sidebar collapses.

https://github.com/Automattic/wp-calypso/assets/402286/9553a4b0-b247-482f-8eee-6a0f0bbf2489

## Testing Instructions

* Go to `/sites?flags=layout/dotcom-nav-redesign-v2`
* Click on a site and check the sidebar transition.
